cityelf/src/main/java/com/cityelf/service/UserService.java

```diff
@@ -5,15 +5,33 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
+import java.util.*;
```

Comment:
Exact class should be used instead of wildcard notation.
import java.util.List; - for example